noir_by_example/README.md

```diff
@@ -0,0 +1,61 @@
+# Examples showcasing syntax, libraries, and data structures
+
+These will show Noir side-by-side against [Rust by example](https://doc.rust-lang.org/rust-by-example/).
+
+The goal is to highlight the similarities and differences in the language and design patterns.
+
+## Tooling
+
+You will need:
+- `nargo` installed [via noirup](https://noir-lang.org/docs/getting_started/installation/)
+- `cargo` installed [via rustup](https://www.rust-lang.org/tools/install) (optional: to see Rust output)
+
+## What is not covered
+
+The surrounding build artifacts (and additional proving/verifying artifacts for Noir) are not explored in this example.
+
+# Usage
+
+The following commands can be run at the top level `noir_by_example`, or withing the specific topic directory.
+Eg: `cd simple_macros`
+
+## Compiling
+
+- `nargo compile`
+- `cargo build`
+
+## Running main
+
+- `nargo execute`
```

Comment:
I see the following error when running this from the `noir_by_example` folder using nargo version 0.33

![image](https://github.com/user-attachments/assets/8e853edd-20ef-40a6-b741-a41a9a958aaa)

Reply:
Updated instructions to run check and add Prover.toml values